src/tests/multihost/ipa/test_misc.py

```diff
@@ -430,3 +430,42 @@ def check_hostname_hash(hash_value=None):
                                     "2014249/2015070"
         assert hashing_true, "Hostnames not hashed"
         assert hashing_false, "Hostnames hashed"
+
+    def test_anonymous_pkinit_for_fast(self, multihost, backupsssdconf):
+        """
+        :title: Allow SSSD to use anonymous pkinit for FAST
+        :id: 4a3ecc11-0d5b-4dce-bd08-5b1f47164b44
+        :customerscenario: True
+        :description:
+         For SSSD to use FAST a Kerberos keytab and service principal must
+         exist. SSSD to be enhanced to allow for the use on anonymous pkinit
+         to create the FAST session.
+        :steps:
+          1. Setup a IPA server/client with default setting.
+          2. Requests anonymous processing using #kinit -n.
```

Comment:
Step 2 is a test that IPA is a correctly setup to handle anonymous PKINIT, 'Requests' sounds like it is a required step, but it is not. The test would pass if you do not call `kinit -n` but it is good to have this step since if this step fails it gives a clear indication that the issue is already in the IPA setup.

Reply:
okay, so can I check the exit status for # kinit?

Reply:
Yes, if `multihost.client[0].run_command('kinit -n')` doesn't do this automatically it makes sense to check the exit status.

Reply:
updated the code to check exit status.